app/javascript/components/Pages/Borrow.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types"
 import { Link } from 'react-router-dom';
 import { Button } from 'reactstrap'
 
-const Borrow = ({ books }) => (
+const Borrow = ({ books, rentals }) => (
```

Comment:
Are you planning to use rentals in here for the dates?

Reply:
We were thinking about implementing it but were having trouble, but we got it on one of the show pages instead. Will remove from here.